src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -804,8 +804,9 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_t,
                               tl_md, worker, params, &config->super.super,
                               &config->rc_mlx5_common, &init_attr);
 
-    self->super.super.config.tx_moderation = ucs_min(config->super.tx_cq_moderation,
-                                                     self->super.tx.bb_max / 4);
+    self->super.super.config.tx_moderation     = ucs_min(config->super.tx_cq_moderation,
+                                                         self->super.tx.bb_max / 4);
+    self->super.super.config.devx_ack_req_freq = config->super.devx_ack_req_freq;
```

Comment:
it is not relevant for DC?

Reply:
As I understood from @yosefe  - only in RC?

Reply:
need also for DC